build-utils/webpack.common.js

```diff
@@ -1,66 +1,60 @@
 const commonPaths = require('./common-paths')
-const webpack = require('webpack')
+const path = require('path')
 const HtmlWebpackPlugin = require('html-webpack-plugin')
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
 
 const config = {
   entry: {
     app: `${commonPaths.appEntry}/index.tsx`,
-    vendor: ['semantic-ui-react']
   },
   output: {
     path: commonPaths.outputPath,
-    publicPath: '/'
+    publicPath: '/',
   },
   resolve: {
-    extensions: [".js", ".jsx", ".json", ".ts", ".tsx"],
+    alias: {
+      'semantic-ui-react$': path.resolve(__dirname, '../src/semanticUiReactReExport'),
+    },
+    extensions: ['.js', '.jsx', '.json', '.ts', '.tsx'],
   },
   module: {
     rules: [
       {
         test: /\.(ts|tsx)$/,
-        loader: "awesome-typescript-loader",
+        loader: 'awesome-typescript-loader',
       },
-      // {
-      //   enforce: "pre",
-      //   test: /\.js$/,
-      //   loader: "source-map-loader",
-      // },
       {
         // Load fonts
         test: /\.woff($|\?)|\.woff2($|\?)|\.ttf($|\?)|\.eot($|\?)|\.svg($|\?)/,
         use: 'url-loader',
       },
       {
-        // Load other files, images etc
-        test: /\.(png|j?g|gif|ico)?$/,
-        use: 'url-loader',
-      }
-    ]
-  },
-  optimization: {
-    splitChunks: {
-      cacheGroups: {
-        styles: {
-          name: 'styles',
-          test: /\.css$/,
-          chunks: 'all',
-          enforce: true
-        },        
-        vendor: {
-          chunks: 'initial',
-          test: 'vendor',
-          name: 'vendor',
-          enforce: true
-        }
-      }
-    }
+        test: /\.(png|svg|jpg|jpeg|gif)$/i,
+        loader: 'file-loader',
+        options: {
+          name: '[path][name].[ext]',
+        },
+      },
+
+      {
+        test: /\.less$/,
```

Comment:
```suggestion
        test: /\.(less|css)$/,
```
I was playing around with another plugin that had its own .css file, and it was giving a compile error until I explicitly added a css handler in webpack config. So would probably be handy to include this here now.

Reply:
bf834f3b4f4250dc30d44a26d0b6635398ebe7e6